sunpy/net/dataretriever/sources/goes.py

```diff
@@ -21,25 +21,27 @@
 
 class XRSClient(GenericClient):
     """
-    Provides access to the GOES XRS fits files archive.
+    Provides access to several GOES XRS files archive.
 
-    Searches for GOES XRS data both on NASA servers prior to re-processed
-    GOES 13, 14 and 15 and on the NOAA archive for > GOES 13.
-    For satellite numbers > 13 the XRSClient searches the NOAA archive, and
-    returns the re-processed science-quality data for GOES 13, 14 and 15, and
-    also the new GOES-R series 16 and 17.
+    For older GOES satellites (7 and below), NASA servers are used.
+    For newer GOES satellites (8 and above), NOAA servers are used.
 
-    Note - the new science quality data have scaling factors removed for 13, 14 and 15
-    and they are not added to GOES 16 AND 17. This means the peak flux will be different to
-    the older version of the data, such as those collected from the NASA servers.
+    For GOES 8-15, the data is the re-processed science-quality data.
 
-    See the following readmes about the data
+    .. note::
 
-    * Reprocessed 13, 14, 15 :
-        https://satdat.ngdc.noaa.gov/sem/goes/data/science/xrs/GOES_13-15_XRS_Science-Quality_Data_Readme.pdf
+        The new science quality data have the scaling factors removed for GOES 8-15
+        and they are not added to GOES 16 AND 17 data products.
+        This means the peak flux will be different to the older version of the data,
+        such as those collected from the NASA servers.
 
-    * GOES-R 16, 17 :
-        https://data.ngdc.noaa.gov/platforms/solar-space-observing-satellites/goes/goes16/l1b/docs/GOES-R_EXIS_XRS_L1b_Science-Quality_Data_ReadMe.pdf
+    See the following readmes about these data:
+
+    * GOES 1 - 7: https://umbra.nascom.nasa.gov/goes/fits/goes_fits_files_notes.txt
+
+    * Reprocessed 8 - 15: https://www.ncei.noaa.gov/data/goes-space-environment-monitor/access/science/xrs/GOES_1-15_XRS_Science-Quality_Data_Readme.pdf
+
+    * GOES-R 16 - 17: https://data.ngdc.noaa.gov/platforms/solar-space-observing-satellites/goes/goes16/l1b/docs/GOES-R_EXIS_XRS_L1b_Science-Quality_Data_ReadMe.pdf
```

Comment:
Do we know when this gets updated for 18? The archive has no data yet. But I saw that SUVI was there.

Reply:
no idea on the timelines for this for 18. I'll ask